project/.travis.yml.twig

```diff
@@ -42,6 +42,7 @@ env:
     - SYMFONY_DEPRECATIONS_HELPER=weak_vendors
     - TARGET=test
     - UPSTREAM_URL=https://github.com/sonata-project/{{ repository_name }}.git
+    - PHPUNIT_VERSION={{ '5.6' in php ? '5.7', '7' }}
```

Comment:
`,` is wrong here, should be `:`.

https://twig.symfony.com/doc/2.x/templates.html#other-operators

Reply:
Good catch 👍

Reply:
Woops

Reply:
Fixed, very nice catch!